pkg/api/vm/formatter.go

```diff
@@ -22,6 +22,8 @@ const (
 	backupVM       = "backup"
 	restoreVM      = "restore"
 	createTemplate = "createTemplate"
+	addVolume      = "addvolume"
+	removeVolume   = "removevolume"
```

Comment:
use camelCase here

Reply:
Indeed we use camelCase conventionally in harvester but KubeVirt API doesn't. Should we follow Harvester's convention and convert thereafter?

Reply:
I would prefer to, and just convert string to small in the method if needed.